alacritty/src/event.rs

```diff
@@ -839,8 +839,15 @@ impl<N: Notify + OnResize> Processor<N> {
                 Event::TerminalEvent(event) => match event {
                     TerminalEvent::Title(title) => processor.ctx.window.set_title(&title),
                     TerminalEvent::Wakeup => processor.ctx.terminal.dirty = true,
-                    TerminalEvent::Urgent => {
-                        processor.ctx.window.set_urgent(!processor.ctx.terminal.is_focused)
+                    TerminalEvent::Bell => {
+                        let _ = processor
+                            .ctx
+                            .config
+                            .bell()
+                            .command
+                            .as_ref()
+                            .map(Self::start_bell_program);
```

Comment:
This formatting seems very unfortunate. Could we maybe extract a `let bell_config = processor.ctx.config.bell();` to make this use only two lines? I don't care how it is achieved, but this seems very wasteful. `if let Some(xxx)` would be another option I suppose, since mapping just to execute is kinda meh anyways.